rest/data/doc.go

```diff
@@ -1,26 +1,28 @@
 /*
-	Adding to the Connector
+Package data provides database access for the rest package
 
-	The Connector is a very large interface that defines how to access the main
-	state of the database and data central to Evergreen's function. All methods of the
-	Connector are contained in the data package in files depending
-	on the main type they allow access to (i.e. all test access is contained in data/test.go).
+Adding to the Connector
 
-	Extending Connector should only be done when the desired functionality cannot be performed
-	using a combination of the methods it already contains or when such combination would
-	be unseemingly slow or expensive.
+The Connector is a very large interface that defines how to access the main
+state of the database and data central to Evergreen's function. All methods of the
+Connector are contained in the data package in files depending
+on the main type they allow access to (i.e. all test access is contained in data/test.go).
```

Comment:
"test" is super vague as to whether it means "go tests" or "test results" or some other tests, so I would prefer to go with a more obvious example.

(i.e. host access is contained in rest/data/host.go)